client/src/theme/components/modals.scss

```diff
@@ -182,6 +182,11 @@ ion-modal.join-by-link-modal::part(content) {
   }
 }
 
+.user-details-modal {
+  --width: 642px;
+  --height: 468px;
+}
```

Comment:
where is it coming from? copy/paste? is there a specific reflexion behind these values?

Reply:
These values are taken from the mockups, I was unsure if setting them explicitly was necessary or not (and now that you mention it, the `height` value is different in the `revoked` mockup so I'm guessing it's not)

Reply:
Yes in lots of cases the modal will fits its content and it will be good enough.
@fabienSvtr a thought on this?